Makefile

```diff
@@ -6,7 +6,7 @@ unit: compose/fakes/fake_client.go
 	SKIP_COMPOSE_API_TESTS=true ginkgo -r
 
 integration:
-	ginkgo --nodes=2 --timeout 30m -r integration_tests
+	ginkgo --nodes=2 --timeout 60m -r integration_tests
```

Comment:
Latest build finished in 22m. Is this just in case Compose gets any slower?

Reply:
I didn't write this, but yes I think so.

Reply:
Yes, I had a timeout when I was running it locally, and we want to make sure we don't leave test resources behind.